plugins/destination/elasticsearch/client/delete_stale.go

```diff
@@ -58,7 +58,7 @@ func (c *Client) DeleteStale(ctx context.Context, msgs message.WriteDeleteStales
 			}
 			req := deletebyquery.NewRequest()
 			req.Query = &q
-			return c.deleteStaleIndex(gctx, msg.TableName, req)
+			return c.deleteStaleIndex(gctx, msg.TableName+"*", req)
```

Comment:
as we write to a `table.Name + "-" + syncTime.Format(tine.DateOnly)` for tables with no PK